From 01b7801a44592dce3b03891fd08b0fb94c555c3f Mon Sep 17 00:00:00 2001
From: Jiping Ma <jiping.ma2@windriver.com>
Date: Tue, 25 Dec 2018 07:51:43 +0000
Subject: [PATCH] arm64: dts: add drive-strength property in dts for
 xilinx-zcu102 platform

In order to support Swissbit SD card, it is need to set the SDIO pin
drvier strength as 2ma explicitly. So, add the drive-strength property.
Other SD cards(For example, SanDisk and Kingston) are also verifyied
when SDIO pin drvier strength is 2ma, they all work fine.

Signed-off-by: Meng Li <Meng.Li@windriver.com>
Signed-off-by: Jiping Ma <jiping.ma2@windriver.com>
---
 arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
index ae34524..0e15116 100644
--- a/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
+++ b/arch/arm64/boot/dts/xilinx/zynqmp-zcu102-revA.dts
@@ -775,6 +775,7 @@ drivers/hwmon/pmbus/Makefile:11:obj-$(CONFIG_SENSORS_MAX20751)  += max20751.o
 			slew-rate = <SLEW_RATE_SLOW>;
 			io-standard = <IO_STANDARD_LVCMOS18>;
 			bias-disable;
+			drive-strength = <2>;
 		};
 
 		mux-cd {
-- 
2.7.4

